donate/templates/pages/core/contributor_support_page.html

```diff
@@ -0,0 +1,96 @@
+                        If you need help with a contribution, fill out this form and we'll get back to you soon.
+                    {% endblocktrans %}</p>
+
+                    <input type=hidden name="orgid" value="{{ orgid }}">
+                    <input type=hidden name="retURL" value="{{ page.full_url }}?submitted=true">
+
+                    <fieldset class="form-item">
+                        <label for="name">{% trans "Contact Name" %}</label>
+                        <input id="name" maxlength="80" name="name" size="20" type="text">
+                    </fieldset>
+
+                    <fieldset class="form-item">
+                        <label for="email">{% trans "Email" %}</label>
+                        <input id="email" maxlength="80" name="email" size="20" type="text">
+                    </fieldset>
+
+                    <fieldset class="form-item">
+                        <label for="00N0B000006X3fw">{% trans "Language" %}</label>
+                        <div class="form-item__wrapper form-custom-select">
+                            <select id="00N0B000006X3fw" name="00N0B000006X3fw" title="Language">
+                                <option value="">--{% trans "None" %}--</option>
+                                <option value="English">{% trans "English" %}</option>
+                                <option value="French">{% trans "French" %}</option>
+                                <option value="German">{% trans "German" %}</option>
+                                <option value="Spanish">{% trans "Spanish" %}</option>
+                                <option value="Polish">{% trans "Polish" %}</option>
+                                <option value="Portuguese">{% trans "Portuguese" %}</option>
+                                <option value="Other">{% trans "Other" %}</option>
+                            </select>
+                        </div>
+                    </fieldset>
+
+                    <fieldset class="form-item">
+                        <label for="type">{% trans "Type" %}</label>
+                        <div class="form-item__wrapper form-custom-select">
+                            <select id="type" name="type">
+                                <option value="">--{% trans "None" %}--</option>
+                                <option value="Problem">{% trans "Problem" %}</option>
+                                <option value="Feature Request">{% trans "Feature Request" %}</option>
+                                <option value="Question">{% trans "Question" %}</option>
+                                <option value="Customer Care">{% trans "Customer Care" %}</option>
+                                <option value="Fundraising">{% trans "Fundraising" %}</option>
+                                <option value="Legal">{% trans "Legal" %}</option>
+                                <option value="Technical Issue">{% trans "Technical Issue" %}</option>
```

Comment:
Remove all options except, including "other" except for the value of "Thunderbird".

Reply:
I'm... not sure I understood that one? =D

Reply:
I want to set the only available submitting value to 'Thunderbird' It's more of an internal classifying field for donor support than an external field. Or maybe we can hide the field?

Reply:
yeah, let's make that a hidden field if there's only one option.